.github/actions/build-binaries/linux/Dockerfile

```diff
@@ -0,0 +1,105 @@
+# Modified from https://github.com/oleksis/pyinstaller-manylinux
```

Comment:
Why? What's different?

Reply:
Looks like you've updated it a bunch and simplified, which is good - I wonder at that point if it's worth even having that link in there? Comparing the two files they're quite different at this point.